helloworld-mutual-ssl-secured/configure-client-truststore.cli

```diff
@@ -0,0 +1,11 @@
+#extract the certificate for the server keystore into a file called serverCert.crt
+/subsystem=elytron/key-store=applicationKS:export-certificate(alias=server, path=serverCert.crt, relative-to=jboss.server.config.dir, pem=true)
```

Comment:
What if the server's certificate hasn't been generated yet? By default, the server's certificate will be automatically generated the first time the HTTPS interface is accessed if it doesn't already exist. If this script is being run during provisioning, I don't think this would exist yet.

Reply:
You can ignore this comment, I see that you're generating the certificate in the next script and that one gets run first :)

Reply:
That is right. I ran into this issue when running the provisioned server, which is why I had to alter the SSL configuration cli.